### ISSUE TITLE
Missing demo for "sliding view"

### ISSUE THREAD

**Issue by OndeVai** [triceam/app-UI#33]
The demo on page http://triceam.github.com/app-UI/examples/samples/03%20-%20slidingView/01%20-%20basic/index.html just shows "body here".

**Comment by rkumar670** [comment]
Yes the whole design concept on the body area. if you add your tag like div or #divbody it will work but some feature will not you can do customize according to your requirement.. check viewnavigator.js :)

**Comment by triceam** [comment]
rkumar670 is correct. That is the demo.  Click/touch and drag on the body to reveal a "sidebar" element underneath of the body.   I can add some more descriptive text, but it is working fine.
